### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by hell0men**
I'am happy to have a chance to contribute to my favorite wallet. The translation was done using AI and carefully checked manually by a native speaker.

### CODE REVIEW

**Comment by vvvvvv1vvvvvv** [comment]
Thanks for contribute, would you like to tell me your twitter id and DeBank web3id so we can mention you in annocement?

**Comment by hell0men** [comment]
Yeah, you may tag https://debank.com/profile/0x4452b58d2fdb19dc757085ef6200a5ff549fbda9

File: _raw/locales/ru/messages.json [hunk 1]
```diff
@@ -0,0 +1,1654 @@
...
+          "doc1": "Подключите один Ledger",
+          "doc2": "Введите пин-код для разблокировки",
+          "doc3": "Откройте приложение Ethereum",
+          "reconnect": "Если это не работает, пожалуйста, попробуйте <1>подключиться сначала</1>."
+        },
+        "howToSwitch": "Как переключить"
+      }
+    },
+    "nft": {
+      "floorPrice": "/ Floor Price:",
```
Comment (vvvvvv1vvvvvv) [inline _raw/locales/ru/messages.json:947]: Is the `/` typo?

**Review by vvvvvv1vvvvvv — APPROVED** [review]

File: _raw/locales/ru/messages.json (hunk 1 above)
Comment (hell0men) [reply _raw/locales/ru/messages.json:947]: Yeah, looks like. Original file has it too.

**Pull request merged**